ingestion/ingestion_server/file_upload/index.html

```diff
@@ -200,7 +202,18 @@ <h3 class="title"></h3>
             <span class="size">{%=o.formatFileSize(file.size)%}</span>
         </td>
         <td>
+            <select style="width:33%;display: inline;" class="form-control" id="select_user" multiple>
```

Comment:
We discussed that this needs to be removed. Since aything under file_upload is an external dependency,  we should not change the html in a way that will cause merge conflicts in the future. Therefore, sharing should be done in another page.